Description: Fix qmin calls
 Add some typecasts so that both parameters to qMin are of the same type
 To fix arm build failure.
Bug-debian: http://bugs.debian.org/??????
Author: Peter Michael Green <plugwash@p10link.net>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- calligra-2.4.2.orig/plugins/pathshapes/rectangle/RectangleShape.cpp
+++ calligra-2.4.2/plugins/pathshapes/rectangle/RectangleShape.cpp
@@ -61,8 +61,8 @@ bool RectangleShape::loadOdf(const KoXml
         QString cornerRadius = element.attributeNS(KoXmlNS::draw, "corner-radius", "");
         if (! cornerRadius.isEmpty()) {
             float radius = KoUnit::parseValue(cornerRadius);
-            m_cornerRadiusX = qMin(radius / (0.5 * size().width()) * 100, qreal(100));
-            m_cornerRadiusY = qMin(radius / (0.5 * size().height()) * 100, qreal(100));
+            m_cornerRadiusX = qMin(qreal(radius / (0.5 * size().width()) * 100), qreal(100));
+            m_cornerRadiusY = qMin(qreal(radius / (0.5 * size().height()) * 100), qreal(100));
         }
     }
 
